tModPorter/tModPorter.Tests/TestData/ModProjectileTest.Expected.cs

```diff
@@ -1,9 +1,17 @@
 using System;
 using Terraria.ModLoader;
+using Microsoft.Xna.Framework; // Vector2, Color
+// Blank
```

Comment:
Yeah as above, we probably won't remove redundant usings. Could be considered helpful in the future? But it's a standard feature of VS